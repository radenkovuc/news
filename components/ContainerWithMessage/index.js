import styled from 'styled-components'
import React from 'react'
import withTranslation from '../HOCs/withTranslation'

const Container = styled.div`
  justify-content: center;
  display: flex;
  flex-direction: column;
  flex: 1;
`

const Text = styled.div`
  text-align: center;
  font-size: 22px;
  font-family: 'Nunito Sans black', sans-serif;
`

type Props = {
  message: String,
  t: Function
}

const ContainerWithMessage = (props: Props) => {
  const {t, message} = props
  return (
    <Container>
      <Text>{t(message)}</Text>
    </Container>
  )
}

ContainerWithMessage.defaultProps = {
  message: '',
  t: (t) => t
}

export default withTranslation(ContainerWithMessage)
