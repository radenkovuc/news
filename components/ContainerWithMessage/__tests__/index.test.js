import React from 'react'
import renderer from 'react-test-renderer'
import ContainerWithMessage from '../index'

describe('<ContainerWithMessage />', () => {
  it('match snapshot', () => {
    const tree = renderer.create(<ContainerWithMessage />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
