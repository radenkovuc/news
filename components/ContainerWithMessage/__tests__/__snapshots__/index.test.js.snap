// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContainerWithMessage /> match snapshot 1`] = `
<div
  className="Container-sc-11nt41q-0 hnURqu"
>
  <div
    className="Text-sc-11nt41q-1 hKYipo"
  >
    
  </div>
</div>
`;
